JSDemos/Demos/Charts/LoadDataOnDemand/description.md

```diff
@@ -1 +1,29 @@
-In&nbsp;this demo, the chart loads data as&nbsp;you pan&nbsp;it. Once a&nbsp;data block is&nbsp;loaded, it&nbsp;stays in&nbsp;memory to&nbsp;reduce requests to&nbsp;the server.
\ No newline at end of file
+In this demo, the [range area](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/RangeArea/) Chart loads data as you pan it. Once the component loads a data block, this data stays in memory to reduce requests to the server. To implement this functionality, do the following:
+
+1. Configure a data source.
+
+    - Assign a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) with an empty [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) to the Chart [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property.
+
+    - Set the [sort](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#sort) property to `date`.
+
+    - Disable the [paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) property to prevent data from partitioning.
+
+2. Configure the Chart to support the data load on demand.
```

Comment:
I know it's an ING, but it's the easiest thing I can do here. I could also say "on-demand data load operations", but it's not as straightforward. 

```suggestion
2. Configure the Chart to support on-demand data loading.
```